karapace/schema_registry_apis.py

```diff
@@ -907,29 +901,3 @@ def no_master_error(self, content_type):
             content_type=content_type,
             status=HTTPStatus.INTERNAL_SERVER_ERROR,
         )
-
-
-def main() -> int:
-    parser = argparse.ArgumentParser(prog="karapace", description="Karapace: Your Kafka essentials in one tool")
-    parser.add_argument("--version", action="version", help="show program version", version=karapace_version.__version__)
-    parser.add_argument("config_file", help="configuration file path", type=argparse.FileType())
-    arg = parser.parse_args()
-
-    with closing(arg.config_file):
-        config = read_config(arg.config_file)
-
-    logging.basicConfig(level=logging.INFO, format=DEFAULT_LOG_FORMAT_JOURNAL)
-    logging.getLogger().setLevel(config["log_level"])
-    kc = KarapaceSchemaRegistry(config=config)
-    try:
-        kc.run(host=kc.config["host"], port=kc.config["port"])
-    except Exception:  # pylint: disable-broad-except
-        if kc.raven_client:
-            kc.raven_client.captureException(tags={"where": "karapace"})
-        raise
-
-    return 0
```

Comment:
If I understand the PR right, this was a secondary way to call Karapace, but we shouldn't have it as we want people to use the main one. Right?

Reply:
Pretty much